chore(coverage): print path relative to snapshot file

Boshen opened the pull request:
Ctrl + click in terminal can open the file with this change.

I don't know about vscode, but maybe a plugin can help?

<img width="1399" alt="image" src="https://github.com/user-attachments/assets/9bb4609b-14e9-4df8-b5ee-cb96b72b2f7d">

<img width="1131" alt="image" src="https://github.com/user-attachments/assets/e0293693-f755-4b91-8712-bbd2a0e615cc">

DonIsaac commented:
Trying to catch back up with @overlookmotel's diffs? 😆 
![image](https://github.com/user-attachments/assets/ecd70394-6528-4ce8-90d7-44b9ed75f835)

Boshen committed:
chore(coverage): print path relative to snapshot file (#4938)

Ctrl + click in terminal can open the file with this change.

I don't know about vscode, but maybe a plugin can help?

<img width="1399" alt="image" src="https://github.com/user-attachments/assets/9bb4609b-14e9-4df8-b5ee-cb96b72b2f7d">

<img width="1131" alt="image" src="https://github.com/user-attachments/assets/e0293693-f755-4b91-8712-bbd2a0e615cc">
